examples/rat/2_prep_lowres_rat.py

```diff
@@ -141,23 +156,32 @@
     padded_flipped_mask = np.flip(padded_mask, axis=2)
 
     # Construct new filename for the flipped image
-    flipped_filename = img_path.stem + "_padded_flipped.nii.gz"
+    flipped_filename = (
+        img_path.with_suffix("").stem + "_N4_padded_flipped.nii.gz"
+    )
     flipped_filepath = flipped_folder / flipped_filename
-    flipped_mask_filename = mask_path.stem + "_padded_flipped.nii.gz"
+    flipped_mask_filename = (
+        mask_path.with_suffix("").stem + "_N4_padded_flipped.nii.gz"
+    )
     flipped_mask_filepath = flipped_folder / flipped_mask_filename
 
     save_as_asr_nii(padded_flipped_img, lowres_vox_sizes, flipped_filepath)
     save_as_asr_nii(
         padded_flipped_mask, lowres_vox_sizes, flipped_mask_filepath
```

Comment:
```suggestion
        padded_flipped_mask.astype(np.uint8), lowres_vox_sizes, flipped_mask_filepath
```
and for any other final masks that you save